data/net/minecraft/SharedConstants.mapping

```diff
@@ -1,12 +1,116 @@
 CLASS net/minecraft/SharedConstants
+	COMMENT Shared global constants.
+	COMMENT
+	COMMENT <p><strong>Note:</strong> The majority of the fields within this class are {@code public static final} with constant expressions (constants), and are inlined by the Java compiler at all places which reference these constant fields. Therefore, changing the value of these constant fields will have no effect on already compiled code.</p>
+	COMMENT
+	COMMENT <p>In addition, it is presumed that a large portion of these constant fields (such as those prefixed with {@code DEBUG_} are used as 'flags', for manually toggling code meant for use by Mojang developers in debugging. Therefore, optimizing compilers (which include the Java compiler) may omit the code hidden behind disabled flags, and will result in these flags having no apparent use in the code (when in reality, the optimizing compiler has removed the code which uses them).</p>
+	COMMENT
+	COMMENT @see <a href="https://docs.oracle.com/javase/specs/jls/se17/html/jls-15.html#jls-15.29">The Java&reg; Language Specification, Java SE 17 Edition, &sect; 15.29. "Constant Expressions"</a>
+	COMMENT @see <a href="https://docs.oracle.com/javase/specs/jls/se17/html/jls-14.html#jls-14.22">The Java&reg; Language Specification, Java SE 17 Edition, &sect; 14.22. "Unreachable Statements"</a>
```

Comment:
Perhaps link to the J16 spec since that's the version Minecraft is currently compiled in.

Reply:
Fair point; I've changed them.